### DISCUSSION TITLE
Help with understanding how piper works

### DISCUSSION THREAD

**Discussion by VladS159** [rhasspy/piper#501] [Q&A] [answered]
Hello, does anyone know what type of architecture PiperTTS uses? Or are there any papers on the subject? I am trying to understand the model behind the result, but I am having quite a hard time. Is it in any way similar to coquiTTS or other similar models? :'(

**Comment by synesthesiam** [comment, marked as answer]
It uses VITS: https://github.com/jaywalnut310/vits/

**Reply by VladS159** [reply]
Thank you so so much and sorry for the, possibly stupid, question!!!!

**Reply by synesthesiam** [reply]
No worries, happy to help 🙂

Also different from Coqui: Piper models are exported to onnx.

**Reply by angad-singhh** [reply]
Hey @VladS159 @synesthesiam , I am relatively new to ML and stuffs, so is their any workflow diagram how does piper work, apart from https://github.com/jaywalnut310/vits/  this, it is little above my understanding. Basically I'm working on a project that will use piper as backend for tts. I wanted some simple workflow to get understanding clear.
